<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>585</width>
      <height>629</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>519</y>
       <width>465</width>
       <height>44</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="datacube">
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>109</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>datacube</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dc_nm">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>32</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>561</y>
       <width>465</width>
       <height>44</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="polarization">
        <property name="minimumSize">
         <size>
          <width>109</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>109</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>polarization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="po_nm">
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="fetch">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>550</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Fetch</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>581</width>
       <height>501</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="mplvl"/>
    </widget>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>590</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="calc">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>570</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>181</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Calculate fields</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>607</x>
      <y>12</y>
      <width>202</width>
      <height>475</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="mplfigs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>311</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="rho_c_t">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>121</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>rho^2 - Crosby</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="rho_c">
         <property name="minimumSize">
          <size>
           <width>151</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>151</width>
           <height>41</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rho_c_t_2">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica</family>
           <pointsize>18</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>rho^2 - Hanson</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="rho_h">
         <property name="minimumSize">
          <size>
           <width>151</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>151</width>
           <height>41</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4"/>
   </widget>
   <widget class="QPushButton" name="res">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <zorder>mplwindow</zorder>
   <zorder>calc</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>fetch</zorder>
   <zorder>progressBar</zorder>
   <zorder>res</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
